content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -1,47 +1,152 @@
-With *Odoo*, you can configure this by using the concept of virtual warehouses. Let us show you how
-to set those virtual warehouses.
+While keeping stock (and selling inventory) from one warehouse might work for smaller companies,
+bigger companies might need to keep stock in (and sell from) multiple warehouses in multiple
+locations.
 
-Set up virtual warehouses
-=========================
+Sometimes, products included in a single sales order might take stock from two (or more)
+:guilabel:`Warehouses`. In Odoo, this can be done by using *virtual locations*.
 
-Let’s say you have two warehouses: Warehouse A and Warehouse B. Create a new warehouse, that will be
-a virtual one. It will allow you to take the stock from A or B. To do so, go to your inventory app
-settings and enable the multi-warehouses feature. Then, go to the warehouses menu and click on
-create.
+.. note::
+   To create virtual locations in warehouses, the :guilabel:`Storage Locations` and
+   :guilabel:`Multi-Step Routes` features will need to be enabled. To do so, go to
+   :menuselection:`Inventory --> Configuration --> Settings`, scroll down to the
+   :guilabel:`Warehouse` section, click the :guilabel:`checkbox` next to those two settings, and
+   :guilabel:`Save` changes.
+
+Create and configure a virtual parent location
+==============================================
+
+Before creating any virtual stock locations, a new warehouse will need to be created. This new
+warehouse will act as a "virtual" warehouse, and will be the "parent" location of other physical
+warehouses.
+
+.. spoiler:: Why a "virtual" warehouse?
+
+   Virtual warehouses are great for companies with multiple physical warehouses. This is because
+   a situation might arise when one warehouse runs out of stock of a particular product, but
+   another warehouse still has stock on-hand. In this case, stock from these two (or more)
+   warehouses could be used to fulfill a single sales order.
+
+   The "virtual" warehouse acts as a single aggregator of all the inventory stored in a company's
+   physical warehouses, and is used (for traceability purposes) to create a hierarchy of locations
+   in Odoo.
+
+Create a new warehouse
+----------------------
+
+To create a new warehouse, go to :menuselection:`Inventory --> Configuration --> Warehouses`, and
+click :guilabel:`Create`. From here, the warehouse :guilabel:`Name` and :guilabel:`Short Name` can
+be changed, and other warehouse details can be changed under the
+:guilabel:`Warehouse Configuration` tab.
 
-.. image:: stock_warehouses/ware1.png
+Under :guilabel:`Shipments`, the number of steps used to process :guilabel:`Incoming Shipments` and
+:guilabel:`Outgoing Shipments` (1 step, 2 steps, or 3 steps) can be configured. Under
+:guilabel:`Resupply`, settings to resupply this warehouse can be configured. This includes whether
+this warehouse should :guilabel:`Manufacture` products, :guilabel:`Buy to Resupply` products, or
+:guilabel:`Resupply Subcontractors`.
```

Comment:
It's not helpful to the reader to reflect back surface-level information they can see for themselves. We can help guide them on their options by covering the form options, thoroughly.

Especially in Odoo 14, these options don't have any tooltips and the information on their use is hard to find on search.
```suggestion
- :guilabel:`Resupply Subcontractors`: resupply subcontractors with components from this warehouse.
- :guilabel:`Manufacture to Resupply`: when products are manufactured, they can be manufactured in
  this warehouse.
- :guilabel:`Manufacture`: to produce right away, move the components to the production location
  directly and start the manufacturing process; to pick first and then produce, unload the
  components from the stock to input location first, and then transfer it to the production
  location.
- :guilabel:`Buy to Resupply`: when products are bought, they can be delivered to this warehouse.
- :guilabel:`Resupply from`: automatically create routes to resupply this warehouse from another
  chosen warehouse
```